docs/index.yml

```
@@ -117,57 +103,67 @@ conceptualContent:
         - url: framework/index.yml
           itemType: overview
           text: .NET Framework (Windows) apps
+        - url: /answers/products/dotnet
+          itemType: get-started
+          text: .NET Q&A
     # Card
     - title: Develop .NET apps
-      summary: Start developing with .NET
       links:
         - url: core/sdk.md
-          itemType: download
-          text: Download the .NET Core SDK
+          itemType: overview
+          text: .NET SDK overview
         - url: core/tools/index.md
           itemType: overview
-          text: .NET Core CLI overview
-        - url: core/porting/index.md
-          itemType: how-to-guide
-          text: Port from .NET Framework to .NET Core
-        - url: core/compatibility/index.md
+          text: .NET CLI overview
+        - url: fundamentals/index.md
           itemType: concept
-          text: .NET Core breaking changes
-        - url: /dotnet/desktop/wpf/overview/
+          text: .NET fundamentals
+        - url: core/porting/index.md
           itemType: overview
-          text: Create .NET Core desktop apps for Windows
-        - url: standard/library-guidance/index.md
+          text: Port from .NET Framework to .NET
+        - url: core/docker/build-container.md
+          itemType: tutorial
+          text: Containerize .NET apps with Docker
+        - url: core/compatibility/index.md
           itemType: concept
-          text: Open source library guidance
+          text: .NET breaking changes
+        - url: /dotnet/desktop/wpf/get-started/create-app-visual-studio?view=netdesktop-5.0
+          itemType: tutorial
+          text: Create .NET desktop apps for Windows
     # Card
     - title: .NET architectural guides
-      summary: Read foundational development and architectural guidance for .NET
       links:
+        - url: architecture/dapr-for-net-developers/index.md
+          itemType: architecture
+          text: Dapr for .NET developers
         - url: architecture/microservices/index.md
           itemType: architecture
-          text: ".NET Microservices: Architecture for containerized .NET apps"
+          text: Architecture for containerized .NET apps
```

```suggestion
          text: .NET Microservices Architecture Guidance
```
How does this sound? I prefer to have the word "Microservices" as people constantly look for this info.

---

I like it, I will update it with this suggestion -- thanks